run_tests.sh

```diff
@@ -0,0 +1,5 @@
+BLACKLIST="allocator-test-NvmCacheTests
+	common-test-TimeTests
+	shm-test-test_page_size"
+
+find -type f \( -not -name "*bench*" \) -executable | grep -vF "$BLACKLIST" | xargs -n1 bash -c
```

Comment:
Will it be possible to show some output of this command so that we can see which tests were running / passing / failing?